lib/tasks/cucumber.rake

```diff
@@ -5,35 +5,35 @@
 # files.
 
 
-unless ARGV.any? {|a| a =~ /^gems/} # Don't load anything when running the gems:* tasks
+unless ARGV.any? { |a| a =~ /^gems/ } # Don't load anything when running the gems:* tasks
 
-vendored_cucumber_bin = Dir["#{Rails.root}/vendor/{gems,plugins}/cucumber*/bin/cucumber"].first
+  vendored_cucumber_bin = Dir["#{Rails.root}/vendor/{gems,plugins}/cucumber*/bin/cucumber"].first
```

Comment:
In general it feels dangerous to make any changes to this file, because it makes it harder to upgrade cucumber and easily see the diff between the new version from the gem and our version.  I wonder if we could disable rubocop for this file?  If we decide to lint it though, the indent added here should be applied to the whole file as it's all inside the `unless` statement on line 8.